tests/functional/test_basic.py

```diff
@@ -651,6 +651,15 @@ def test_several_scripts2(pyi_builder_spec):
     pyi_builder_spec.test_spec('several-scripts2.spec')
 
 
+def test_spec_with_hyphenated_hiddenimport(pyi_builder_spec):
+    """
+    Verify that a spec whose hiddenimports include a hyphenated module name is valid
+
+    See issue #8591
+    """
+    pyi_builder_spec.test_spec('spec_with_hyphenated_hiddenimport.spec')
```

Comment:
This test does not need a spec (and should not be using scripts from unrelated tests):

```python
def test_hyphenated_hiddenimport(pyi_builder):
    """
    Verify that a spec whose hiddenimports include a hyphenated module name is valid
    See issue #8591
    """
    pyi_builder.test_source(
        """
        print("hello!")
        """, pyi_args=['--hiddenimport', 'fake-hyphenated-module']
    )
```

Reply:
Ah, thanks. I looked for a way to do that, but obviously did not look hard enough. Appreciate the pointer to the right affordance.